__tests__/app.test.js

```diff
@@ -30,16 +31,47 @@ describe('GET/api/topics', () => {
                 expect(topic).toHaveProperty('description')
             })
         })
-        
+    })
+})
+describe('GET/api', () => {
+    test('200: should respond with an object describing all available endpoints', () => {
+        return request(app)
+        .get('/api')
+        .expect(200)
+        .then(({body}) => {
+            const resEndpoints = body
+            expect(resEndpoints).toEqual(endpoints)
```

Comment:
This test is excellent 🏅 
I'm not sure about line 42 because it's the same object just being renamed 🤷